interface/repository/pokemon_repository.go

```diff
@@ -0,0 +1,67 @@
+package repository
+
+import (
+	"encoding/csv"
+	"io"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/Topi99/academy-go-q12021/domain/model"
+	"github.com/Topi99/academy-go-q12021/usecase/repository"
+)
+
+type pokemonRepository struct {
+}
+
+// PokemonRepository interface
+type PokemonRepository interface {
+	FindOne(id uint) (*model.Pokemon, error)
+}
+
+// NewPokemonRepository returns new PokemonRepository
+func NewPokemonRepository() PokemonRepository {
+	return &pokemonRepository{}
+}
+
+func (po *pokemonRepository) FindOne(id uint) (*model.Pokemon, error) {
+	absPath, _ := filepath.Abs("infrastructure/datastore/pokemons.csv")
+	f, err := os.Open(absPath)
+
+	if err != nil {
+		return nil, err
+	}
+
+	r := csv.NewReader(f)
```

Comment:
I don't know if reading from the CSV file should be in the repository file, or in other file inside `infrastructure` module.